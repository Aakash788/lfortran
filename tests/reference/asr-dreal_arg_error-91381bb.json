{
    "basename": "asr-dreal_arg_error-91381bb",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/dreal_arg_error.f90",
    "infile_hash": "b3eef179f8004efffebeb00ee2a7cccd1d2c6c90cfaa04973a3e8d50",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-dreal_arg_error-91381bb.stdout",
    "stdout_hash": "2fb2f7bc01ef79953f830ab2354434e5d731e01cfe9697dd72818e9f",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}