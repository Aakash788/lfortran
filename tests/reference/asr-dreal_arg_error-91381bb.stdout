(TranslationUnit
    (SymbolTable
        1
        {
            dreal_arg_error:
                (Program
                    (SymbolTable
                        2
                        {
                            a:
                                (Variable
                                    2
                                    a
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Complex 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    dreal_arg_error
                    []
                    [(Assignment
                        (Var 2 a)
                        (ComplexConstructor
                            (RealConstant
                                1.000000
                                (Real 4)
                            )
                            (RealConstant
                                2.000000
                                (Real 4)
                            )
                            (Complex 4)
                            (ComplexConstant
                                1.000000
                                2.000000
                                (Complex 4)
                            )
                        )
                        ()
                    )
                    (Print
                        [(IntrinsicElementalFunction
                            Dreal
                            [(Var 2 a)]
                            0
                            (Real 8)
                            ()
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
